modules/gltf/structures/gltf_node.h

```diff
@@ -96,6 +97,10 @@ class GLTFNode : public Resource {
 
 	GLTFLightIndex get_light();
 	void set_light(GLTFLightIndex p_light);
+
+	Variant get_additional_data(const String &p_extension_name);
```

Comment:
Does it matter if it's string or stringname?

Reply:
Either works as long as it's consistent. I think that StringName is better. Changed.